app/Http/Controllers/Store/BundleController.php

```diff
@@ -282,6 +291,11 @@ public function redirectAfterRequest($errors, $successRoute, $failRoute)
                     case "foreign":
                         $messages[] = "Action denied on a foreign voucher: " . join(', ', $values);
                         break;
+                    case "empty":
```

Comment:
There might be an alternative to relying on keyed bools for error string generation. But happy to keep going with this for now.